src/sync/sync.rs

```diff
@@ -72,6 +74,7 @@ mod platform {
     }
 
     unsafe fn find_first_volume() -> (String, winnt::HANDLE) {
+        #[allow(deprecated)]
```

Comment:
Again, I think you should use `MaybeUninit` here.

Reply:
... per notes above